package.json

```diff
@@ -29,6 +29,9 @@
     "type": "git",
     "url": "https://github.com/mjeanroy/rollup-plugin-license"
   },
+  "peerDependencies": {
+    "rollup": "1.1.0"
```

Comment:
Currently, this plugin is compatible with all versions of rollup, so the version here should probably be: `< 2`.